src/plugins/intel_gpu/src/kernel_selector/core/actual_kernels/range/range_kernel_selector.h

```diff
@@ -4,7 +4,7 @@
 
 #pragma once
 
-#include "kernel_selector.h"
+#include <kernel_selector.h>
```

Comment:
why range is included from this PR?

Reply:
Seems that this is only a small refactoring of another operation and no real logic is changed, I have reverted this.